start-with-rest/supergraph.yaml

```
@@ -1,8 +1,8 @@
 # A federated GraphQL API (aka a supergraph) is a unified GraphQL API composed of different services—known as subgraphs—that are managed through a router.
 
-federation_version: 2.10.0
-
-subgraphs: # Use this section to add all of the services that make up your GraphQL API
-  products: # The subgraph name. Populate this with your service's name.    routing_url: http://placeholder # This value is ignored
+federation_version: =2.10.0
+subgraphs: # Use this section to add all of the services that make up your GraphQL API 
```

I would normally prefer "graph" here, but since we are already explaining what a "supergraph" is, let's stick to that term.

```suggestion
subgraphs: # Use this section to add all of the services that make up your supergraph
```

---

One thing I’m wondering is, should we even keep the `supergraph.yaml ` in the repo? rover init actually generates one based on the schemas in the project, so this file will end up getting replaced anyway during the final step. Might be cleaner to leave it out?